mathics/core/element.py

```diff
@@ -369,41 +245,14 @@ def equal2(self, rhs: Any) -> Optional[bool]:
             return self == rhs
         return None
 
-    def evaluate(self, evaluation) -> "BaseElement":
-        """
-        Evaluates the element.
-        Each subclass should decide what is right here.
-        """
-        raise NotImplementedError
-
-    # FIXME the fact that we have to import all of these symbols means
-    # modularity is broken somehwere.
-    # And format really doesn't belong here.
-    def format(self, evaluation, form, **kwargs) -> "BaseElement":
-        """
-        Applies formats associated to the expression, and then calls Makeboxes
-        """
-        from mathics.core.symbols import (
-            Symbol,
-            SymbolMakeBoxes,
-        )
-        from mathics.core.atoms import String
-        from mathics.builtin.base import BoxExpression
-        from mathics.builtin.box.inout import _BoxedString
+    # Deprecated: use mathics.core.formatter.format_element
```

Comment:
Is using `element.format()` deprecated?

Reply:
At the beginning of this, I thought so. Now, I am not sure.

Reply:
I'd say remove the comment.